freedomofpress/securedrop-workstation
# Renames sd-journalist to sd-proxy

Closes #138 
:rotating_light: This requires changes in securedrop-client which are not live, merging this branch must be coordinated with the release of securedrop-client as to not break master. This is because securedrop-sdk hardcodes the proxyvm name, and it must be changed from sd-journalist to sd-proxy (see https://github.com/freedomofpress/securedrop-sdk/pull/43)

Test plan:
1. Do not use this branch; check out master and make clone
2. Make clean (to remove old VMs such as sd-journalist)
3. Check out this branch and make clone
4. Make all
5. Make test
6. Sideload securedrop-client 0.0.5

- [ ] Make test does not return any errors
- [ ] There are no sd-journalist or sd-journalist-template VMs present
- [ ] Documents download and decrypt in sd-client

Merge this at the same time as SecureDrop-client 0.0.5 is deployed to apt-test-qubes
